CHANGELOG.md

```diff
@@ -4,6 +4,48 @@
 
 ### New features
 
+#### Check checkboxes by using the `checkedValues` option
+
+When using the Checkboxes nunjucks macro you can now use `checkedValues` option to indicate which checkboxes should be checked when the page loads.
+
+For example, this would check the checkboxes that have the values 'foo' and 'bar':
+
+```
+checkedValues: ['foo', 'bar']
+```
+
+This is an alternative to setting the `checked` option on each individual checkbox item.
+
+This change was introduced in [pull request #2616: Allow selecting options by passing current values](https://github.com/alphagov/govuk-frontend/pull/2616).
+
+#### Check a radio button by using the `checkedValue` option
+
+When using the Radios nunjucks macro you can now use `checkedValue` option to indicate which radio should be checked when the page loads.
+
+For example, this would check the radio that has the values 'foo':
+
+```
+checkedValue: 'foo'
+```
+
+This is an alternative to setting the `checked` option on each individual radio item.
+
+This change was introduced in [pull request #2616: Allow selecting options by passing current values](https://github.com/alphagov/govuk-frontend/pull/2616).
+
+#### Select an option in a Select by using the `selectedValue` option
```

Comment:
THIS IS A NIGHTMARE. Two of these words have two different meanings and we use both meanings in the same heading.

**Select** (verb) an **option** (an `<option>` HTML element within a `<select>`) in a **Select** (component name / HTML element name) using the `selectedValue` **option** (a 'Nunjucks macro option')

Suggestions welcome.

Reply:
Tricky one alright! Ok, first pass:

> Choose an option in a select component by using the `selectedValue` parameter

Reply:
It might be to save the user time, or more likely it might be because the user has gone back to change an answer they previously gave.

For example, if a user was in the middle of a transaction and went 'back' to a previous page that used a select, the answer they previously gave should be selected when the page loads.

Likewise, if the user was [checking their answers](https://design-system.service.gov.uk/patterns/check-answers/) and decided to change an answer they previously gave, when they click change the page that loads should pre-select the answer that they'd previously given.